docs/references/functions/create-build.md

```diff
@@ -1 +0,0 @@
-Create a new build for an Appwrite Function deployment. This endpoint can be used to retry a failed build.
```

Comment:
Something feels off. This file deletion seems weird, and description says retry so maybe it was incorrect already